src/evaluations/data/evaluation_configs.py

```diff
@@ -115,9 +170,10 @@ def _generate_configs_scenario_3b(universe_size, num_sets, small_set_size,
     user_activity_assciation: user activity association used in the Exponential
       Bow model. Should be one of the defined user activity association defined
       by the set_generator.USER_ACTIVITY_ASSOCIATION_XXX.
-
+      For 3(a) user_activity_assciation=set_generator.USER_ACTIVITY_ASSOCIATION_XXX
```

Comment:
Do you mean USER_ACTIVITY_ASSOCIATION_INDEPENDENT?

Reply:
Nice catch!